src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -162,6 +171,29 @@ function ReportPreview({
         [chatReport?.isOwnPolicyExpenseChat, policy?.harvesting?.enabled],
     );
 
+    const confirmPayment = (type?: PaymentMethodType | undefined) => {
+        if (!type) {
+            return;
+        }
+        setPaymentType(type);
+        setRequestType(CONST.IOU.ACTION_TYPE.PAY);
+        if (ReportUtils.hasHeldExpenses(iouReport?.reportID)) {
+            setIsHoldMenuVisible(true);
+        } else if (chatReport) {
```

Comment:
```suggestion
        } else if (chatReport && iouReport) {
```

we should not disable the lint rule. How about this suggestion?